data/src/emulator.js

```diff
@@ -7174,5 +7174,51 @@ class EmulatorJS {
 
         return recorder;
     }
+
+    enableSaveUpdateEvent() {
+        if (!window.crypto.subtle) {
+            console.error("Attempt to compare save files without subtle crypto (maybe you are running in an insecure context?");
+            return;
+        }
+
+        // https://stackoverflow.com/questions/9267899/
+        function transformDigestToBase64(buffer) {
```

Comment:
Is it possible we could compare actual string hashes intead of base64? something like this:

```javascript
  const hashArray = Array.from(new Uint8Array(buffer));
  const hash = hashArray.map(byte => byte.toString(16).padStart(2, '0')).join('');
```